### DISCUSSION TITLE
I always get "cache miss" in Vercel's build.

### DISCUSSION THREAD

**Discussion by gsphyo91** [vercel/turborepo#1674] [Help] [answered]
The repository consists of 3 apps.
I have created 3 projects in  Vercel, and they are linked to each app directory on Github. (apps/A, apps/B, apps/C)

Executed `npx turbo login`, `npx turbo link`, and executed `vercel link` in each app directory.

When I changed one app and pushed it to github, I expected that only one app would show "cache miss", and the build would work for two apps through the cache.

However, "cache miss" appears in all 3 apps.

How do I make the cache work well?

**Comment by anthonyshew** [comment, marked as answer]
I'm unable to reproduce this so going to close due to age and inactivity. If you're still having this problem today, please open an issue with a reproduction.
